src/api/deploy/helpers/lookup-tenant-service.js

```diff
@@ -31,37 +31,7 @@ async function lookupTenantService(service, environment, ref = 'main') {
     Joi.assert(service, schema) // Check file in correct format
     return service
   } catch (error) {
-    logger.error(
-      `Error attempting to retrieve ${filePath} from GitHub - Falling back to tenant_services.json, ${error}`
-    )
-    return await lookupLegacyTenantService(service, environment, org, repo, ref)
-  }
-}
-
-/**
- * Get service from the legacy single-file version of tenants.json
- * @param {string} service
- * @param {string} environment
- * @param {string} owner
- * @param {string} repo
- * @param {string} ref
- * @returns {Promise<undefined|*>}
- */
-async function lookupLegacyTenantService(
-  service,
-  environment,
-  owner,
-  repo,
-  ref
-) {
-  const filePath = `environments/${environment}/resources/tenant_services.json`
-  logger.info(`Getting legacy tenant file ${filePath}`)
-  try {
-    const data = await getContent(owner, repo, filePath, ref)
-    const services = JSON.parse(data)
-    return services[0][service]
-  } catch (error) {
-    logger.error(error, `Error attempting to retrieve ${filePath} from GitHub`)
+    logger.error(`Error attempting to retrieve ${filePath} from GitHub`)
     return undefined
```

Comment:
FYI if you just `return` this is same same as `return undefined`

Reply:
updated